addons/l10n_hr_edi/models/account_move_send.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import pytz
+
+from datetime import timedelta
+
+from odoo import api, fields, models
+from odoo.addons.l10n_hr_edi.models.mojeracun_connection import MojEracunServiceError
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMoveSend(models.AbstractModel):
+    _inherit = 'account.move.send'
+
+    @api.model
+    def _check_move_constrains(self, moves):
+        if any((move.country_code == 'HR' and not move.l10n_hr_operator_name) for move in moves):
```

Comment:
Dubious. operator_name et operator_oib really defined on account.move? why isn't it a setting, for example?